vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
+
+{{< pagebreak >}}
+
+# Data Conformance Summary
+
+## Conformance Inputs
+
+Were the analysis datasets evaluated for conformance with CDISC ADaM Validation Checks?
+
+```         
+  Yes, Version of CDISC ADaM Validation Checks and software used: Pinnacle 21® 
+  Community 4.0.2
+```
+
+Were the ADaM datasets evaluated in relation to define.xml?
+
+```         
+  Yes
+```
+
+Was define.xml evaluated?
+
+```         
+  Yes                     
+```
+
+## Issues Summary
+
++----------+------------------------------------------------------------------------------+---------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Check ID | Diagnostic Message                                                           | Dataset | Count (Issue Rate) | Explanation                                                                                                                                                                             |
++==========+==============================================================================+=========+====================+=========================================================================================================================================================================================+
+| AD1012   | Secondary custom variable is present but its primary variable is not present | ADSL    | 1 (50.00%)         | This is a Sponsor Extension to the ADaM Model. The VISNUMEN \[End of Trt Visit (Vis 12 or Early Term.)\] variable is a integer variable which is not related to any character variable. |
++----------+------------------------------------------------------------------------------+---------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+: {tbl-colwidths="\[17,22,15,22,25\]"}
+
+
+## QC Findings and Common Issues
+
+In this Pilot 3 study, our focus was to create ADaMs based on Pilot 1 where after ADaM generation we compared them against the analysis datasets used in Pilot 1 as a QC step. With these comparisons we listed the QC Findings with explanations as to why these findings exist. We also came across common issues throughout the ADaM generation process, which could be helpful for improvements utilising the CDISC Pilot data in the future. More details can be found in the appendix (Appendix 2 and Appendix 3).
+
+[QC Findings](https://github.com/RConsortium/submissions-pilot3-adam/wiki/QC-Findings)
```

Comment:
I think we should provide some on text on where this is housed and why it is housed on GitHub

Reply:
Alternatively, if there is a way we can bring in this wiki into the adrg markdown so that it includes it in the adrg, then we won't have to explain why it's housed in Github.

Reply:
I removed to links to github wiki in the adrg and brought in the content directly into the adrg instead.